src/prospectScraper/MockDraftPick.cs

```diff
@@ -143,8 +127,8 @@ public static string GetState(string school)
                 schoolsAndConferences = csv.GetRecords<School>().ToList();
             }
             var stateResult = from s in schoolsAndConferences
-                              where s.schoolName == school
-                              select s.state;
+                              where s.SchoolName == school
+                              select s.State;
 
             string? srfd = stateResult.FirstOrDefault();
```

Comment:
I see this error:
The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.

Should we add this annotation somewhere?